docs/python_docs/python/Makefile_sphinx

```diff
@@ -18,8 +18,25 @@
 # Makefile for Sphinx documentation
 #
 
+
+NPROCS := 1
+OS := $(shell uname)
+export NPROCS
+
+ifeq ($J,)
```

Comment:
can we change J to numjobs or something different than 1 character variable?

Reply:
not sure... is it really important? I'd rather not suffer through another CI pass...

Reply:
yes, it's important. code is usually written once and read by many people many times. having a good variable name will help make their life easier.

Reply:
Ok, please re-review; I updated the variable name.

Reply:
Well, [that change broke it](http://jenkins.mxnet-ci.amazon-ml.com/job/mxnet-validation/job/website/job/PR-16512/2/display/redirect). I think "J" was probably required. IDK enough about it though...

Reply:
environment variable names in makefile require parentheses around them

Reply:
or {}

Reply:
https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html

Reply:
ok this works now!